app/views/stories/_main_stories_feed.html.erb

```diff
@@ -65,11 +11,11 @@
       <% if !user_signed_in? && i == 4 %>
         <%= render "stories/sign_in_invitation" %>
       <% end %>
-      <%= render "articles/single_story", story: story %>
+      <%= render "articles/single_story", story: story, featured: false %>
```

Comment:
https://github.com/thepracticaldev/dev.to/pull/7151/files#r406101990 - same here...